### PR TITLE
Add showing icon on external folders

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Needed for https://github.com/nextcloud/android/pull/2206

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

File: src/com/owncloud/android/lib/resources/files/RemoteFile.java [hunk 1]
```diff
@@ -54,6 +54,7 @@
     private long mSize;
     private boolean mIsFavorite;
     private boolean mIsEncrypted;
+    WebdavEntry.MountType mMountType;
```
[the remark this answers is not in the document]
Comment (AndyScherzinger) [reply src/com/owncloud/android/lib/resources/files/RemoteFile.java]: should be private :)

File: src/com/owncloud/android/lib/common/network/WebdavEntry.java [hunk 2]
```diff
@@ -218,6 +224,19 @@ public WebdavEntry(MultiStatusResponse ms, String splitElement) {
                 mIsEncrypted = false;
             }
 
+            // NC mount-type property <nc:mount-type>
+            prop = propSet.get(EXTENDED_PROPERTY_MOUNT_TYPE, Namespace.getNamespace(NAMESPACE_NC));
+            if (prop != null) {
+                String mountTypeValue = (String) prop.getValue();
+                if ("external".equals(mountTypeValue)) {
```
Comment (AndyScherzinger) [inline src/com/owncloud/android/lib/common/network/WebdavEntry.java]: could be shortened to ```if ("external".equals((String) prop.getValue())) {```

**Comment by AndyScherzinger** [comment]
:+1: Just two minor comments

**Comment by tobiasKaminsky** [comment]
Accidently merged and reverted, so please use #131

**Pull request closed without merging**